src/main.py

```diff
@@ -1,6 +1,12 @@
 from kraken_bot import kraken_bot
 from telegram_manager import send_telegram_message
 from time import sleep
+from orm import Base, engine
+
+# the easiest way to create tables
+# but better to use https://alembic.sqlalchemy.org/
+# so, it store migration story as well
+Base.metadata.create_all(bind=engine)
```

Comment:
Base.metadata.create_all() should be used only in cases when you start with new database.

If you have a database with active strategies and different version of models.py - main.py will return Error on this step and won't run.

Reply:
no, it depends on if you create tables yourself with meta that differ from declared in `models`